### DISCUSSION TITLE
Add markdown code portion counting

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#30976] [Other Feature Feedback, Questions, & Ideas] [answered]
I would really like to see Markdown in the code-portion-counting thing, so that it something like this:
![grafik](https://user-images.githubusercontent.com/50182241/186998978-f24c048a-9227-43c2-a4db-e9fc7e906623.png)
Because currently it just looks like this:
![grafik](https://user-images.githubusercontent.com/50182241/186999057-3e9c832e-f055-4007-9a9c-942d46eea511.png)

I would see it as a feature you could turn on or off in the settings, because in my opinion it's part of something like a docs-repo but should not be there for a repo where the code is the important thing.

Example:   
php/doc-de: Should show Markdown, because it is a repo for just documentation   
supercool123/minecraftMod3000: Should not show Markdown in the Statistics, because it is a mainly for code

**Comment by airtower-luna** [comment, marked as answer]
You should be able to do that already, with "should" meaning "I haven't tried myself, but I've seen it in the docs." GitHub uses Linguist to detect file types, and by default Linguist tries to exclude documentation (and certain other things) from language statistics. But there are overrides you can use if you want to, to make it exclude stuff it wouldn't by default, or (like you want) *not* exclude certain things.

See: https://github.com/github/linguist/blob/master/docs/overrides.md

**Reply by CuzImBisonratte** [reply]
Thank you very much :D
